helm-chart/values.yaml

```diff
@@ -7,13 +7,14 @@ replicaCount: 1
 image:
   repository: supervisions/itop
   pullPolicy: Always
-  tag: 3.0
+  tag: 3.1.0
 
 environment:
-  db_host: <mysql-server-url> "mysql-chart.database.svc.cluster.local"
-  db_name: <mysql-db-name>
-  db_pwd: <mysql-db-pwd>
-  db_user: <mysql-db-user>
+  db_host: itop-mariadb # <address>:<port>
+  db_name: itop_db
+  db_pwd: itop_password
+  db_user: itop_user
```

Comment:
Is it possible to directly use the values from `mariadb.auth` instead?

Reply:
I don't understand. Can it then be the other way round? So that these would be used for `mariadb.auth`? I would like to avoid having the DB credentials defined twice.

Furthermore, these values are only used for initial setup, do they really need to be here in that case? Can the `initContainers` do anything for that?

Reply:
I dont know another approaches. This is standard method if used dependency chart.